src/main/java/athleticli/commands/sleep/AddSleepCommand.java

```diff
@@ -15,6 +16,7 @@ public class AddSleepCommand extends Command {
 
     private LocalDateTime from;
     private LocalDateTime to;
+    private final Logger LOGGER = Logger.getLogger(AddSleepCommand.class.getName());
```

Comment:
Since it is not a constant, maybe it is better to call it `logger` instead of `LOGGER`?